engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -427,7 +428,8 @@ class PeriodicProcessService(
           )
         }
       }
-      .sortBy(_.runAt)(Ordering[LocalDateTime].reverse)
+      .sortBy(_.runAtWithCreatedAt)
+      .reverse
```

Comment:
it looks like the approach with "reversing" the ordering is a little bit efficient (we don't reverse the whole collection but sort with the final order.

Maybe let's create a private val reversedLocalDataTimeOrdering and reuse them all over the class? WDYT?

Reply:
That is true, I've changed it to use reversed ordering instead of reverse sort with standard ordering.